### DISCUSSION TITLE
How to define border width?

### DISCUSSION THREAD

**Discussion by adaboese** [chakra-ui/panda#1953] [Q&A] [answered]
I am trying to use strict tokens, but I cannot figure out how to define allowed values for border width.

I tried adding:

```ts
sizes: {
  '1px': {
    value: '1px',
  },
},
```

but it still does not allow `borderWidth: '1px'`

This is how I try to reference it:

```ts
const TableBox = styled('div', {
  base: {
    borderColor: 'border.1',
    borderRadius: '8px',
    borderWidth: '1px',
    overflow: 'hidden',
  },
});
```

Documentation has this example, but I don't understand what's the relationship to just defining `borderWidth`.

```ts
const theme = {
  tokens: {
    borders: {
      // string value
      subtle: { value: '1px solid red' },
      // string value with reference to color token
      danger: { value: '1px solid {colors.red.400}' },
      // composite value
      accent: { value: { width: '1px', color: 'red', style: 'solid' } }
    }
  }
}
```

**Comment by anubra266** [comment, marked as answer]
@adaboese there's a `borderWidths` token https://play.panda-css.com/GJtrwOSzqw?tab=config
It works but the types were not in panda yet. [Here's](https://github.com/chakra-ui/panda/pull/1959) a PR to add it to the Panda types.

**Reply by adaboese** [reply]
Ah, that makes. I was looking at the the types to understand the available values. Thank you
